docs/en/Configuring-Alluxio-with-OrangeFS.md

```diff
@@ -9,11 +9,20 @@ priority: 2
 This guide describes how to configure Alluxio with [OrangeFS](http://www.orangefs.org/) as the 
 under storage system.
 
-# Initial Setup
-
-First, the Alluxio binaries must be on your machine. You can either
-[compile Alluxio](Building-Alluxio-Master-Branch.html), or
-[download the binaries locally](Running-Alluxio-Locally.html).
+# Building Alluxio-OrangeFS Distro
+
+The integration of Alluxio-OrangeFS depends on 
+[orangefs-jni](http://www.orangefs.org/fisheye/orangefs/browse/orangefs/branches/maven-repository/maven2/org/orangefs/usrint/orangefs-jni) 
+for an optimised I/O performance that is comparable to OrangeFS 
+[native performance](http://docs.orangefs.com/v_2_9/Design_Overview.htm).
+
+Due to licensing issues with runtime proprietary binaries, we do not include the `orangefs-jni` 
+dependency by default. To build an integrated distro, you need to either include the 
+orangefs-jni.jar that comes with 
```

Comment:
I'm thinking it might be a good thing to offer a single building solution instead of two options. Because maven does not support local jar file as a dependency inherently. To do so, users have to create their own maven local repository, which makes unnecessary complexity on the whole setup.